rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1185,7 +1188,8 @@ def _combine_sparse_dense_features(
             else:
                 dense_features.append(f)
 
-        return tf.concat(dense_features, axis=-1) * mask
+        outputs = tf.concat(dense_features, axis=-1) * mask
+        return self._tf_layers[f"dropout.{name}"](outputs)
```

Comment:
Just to make sure, we also want to apply dropout if we only use dense features?

Reply:
yes, I think, it makes sense to do